lib/utilshare/accounts/dwolla.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Utilshare.Accounts.Dwolla do
+
+  def token do
+    GenServer.call(Utilshare.TokenManager, {:get, :token})
+  end
+
+  def authenticate_server() do
+    body = {:form, 
+    [
+      client_id: Application.get_env(:utilshare, :key),
+      client_secret: Application.get_env(:utilshare, :secret),
+      grant_type: "client_credentials",
+    ]}
+    case HTTPoison.post("https://sandbox.dwolla.com/oauth/v2/token", body, %{"Content-type" => "application/x-www-form-urlencoded"}) do
```

Comment:
Ignore if this works for you, but the lower-case `t` in `Content-type` seems like it may cause a bug